src/test_utils/public/enzyme_helpers.tsx

```diff
@@ -60,13 +59,20 @@ function nodeWithIntlProp(node) {
  *  @param options properties to pass into shallow wrapper
  *  @return The wrapper instance around the rendered output with intl object in context
  */
-export function shallowWithIntl(node, { context, childContextTypes, ...props } = {}) {
+export function shallowWithIntl(
```

Comment:
```suggestion
export function shallowWithIntl<T>(
  node: ReactElement<T>,
```
Let's preserve type where we can.